Sources/PackageDescription/Action.swift

```diff
@@ -0,0 +1,47 @@
+/// Describes what should be run
+///
+/// - custom: is run from root of your library
+/// - predefined: Runs one of `PredefinedAction`s that is provided by tapestry
+public enum Action: Codable {
+    case custom(tool: String, arguments: [String])
```

Comment:
❕ I think that doc comments should rather be tied to cases then mentioned in doc comments of the `enum` or maybe both 🙂

Reply:
Yeah, left them in both - more documentation will do no harm 🙂